vignettes/Introduction.Rmd

```diff
@@ -120,11 +120,9 @@ q$remove()
 
 However, private members can't be accessed directly:
 
-```{r eval = FALSE}
+```{r, error=TRUE}
```

Comment:
By evaluating this code chunk, the output here for `q$length()` would change from:

```R
q$length()
#> Error: attempt to apply non-function
```
to :

```R
q$length()
#> Error in eval(expr, envir, enclos): attempt to apply non-function
```

Which I think is a step backward for clarity.

Reply:
I have reverted changes to all relevant code blocks.

> I think is a step backward for clarity

Sure, the error message might be clearer when hard-coded, but they might also be confusing for some who might wonder why they don't get the exact same error as described in the vignette.

Additionally, if the underlying error description changes in base-R itself, there is no way for the vignette to catch this because the error description is hard-coded.

This is why I had made those changes originally.

Reply:
Just to be clear, the hard-coded error message _is_ the same message that R gives at the console.

<img width="281" alt="image" src="https://user-images.githubusercontent.com/86978/156455172-4973022b-a423-4e10-9fe0-eea3adf93061.png">

The message with `Error in eval...` is happens when knitr processes the code chunks.

I suppose it's possible that base R could change the error message at some point in the future, but it seems very unlikely to me.

Reply:
Okay, fair enough! 

I have already reverted all changes to code chunks with `eval=FALSE`.